config/schema/format_strawberryfield.schema.yml

```diff
@@ -308,6 +308,11 @@ field.formatter.settings.strawberry_paged_formatter:
       type: string
     manifesturl_source:
       type: string
+    metadataexposeentity_source:
```

Comment:
Once I figured out that upper/lower case matters in the `hascover` field name, I confirmed that setting it to false, using the 'paged' view hint, I get two pages displaying side by side in the default view. :+1: